src/content/docs/kubernetes-pixie/k8s-otel/install.mdx

```diff
@@ -0,0 +1,612 @@
+---
+title: Install OpenTelemetry Collector on Kubernetes
+tags:
+    - Kubernetes integration
+    - New Relic K8S OpenTelemetry integration
+    - New Relic OpenTelemetry Collector
+    - NRDOT
+    - New Relic OpenTelemetry
+    - OpenTelemetry Collector
+    - OpenTelemetry Kubernetes
+    - OpenTelemetry Helm chart
+    - New Relic OpenTelemetry installation
+metaDescription: "Instrument your Kubernetes Cluster in New Relic using OpenTelemetry Collectors"
+freshnessValidatedDate: 2024-07-23
+---
+
+You can monitor your Kubernetes cluster using OpenTelemetry, which provides a standardized way to collect and send telemetry data to New Relic. This integration allows you to gain insights into your Kubernetes environment, including metrics, events, and logs. 
```

Comment:
Should we include a link to NRDOT ?
https://github.com/newrelic/nrdot-collector-releases/tree/main/distributions/nrdot-collector-k8s